src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseActivity.md

```diff
@@ -46,6 +55,19 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.```yaml
```

Comment:
@jiawang003 this is a bug when using `platyPS`, but make sure the ` ``yaml ` string at the end of this line is moved on a line by itself. See the other parameters in this markdown file as an example of how this should look.

Reply:
fixed

Reply:
fixed